templates/AureliaSpa/webpack.config.js

```diff
@@ -1,24 +1,28 @@
-var isDevBuild = process.argv.indexOf('--env.prod') < 0;
-var path = require('path');
-var webpack = require('webpack');
-var AureliaWebpackPlugin = require('aurelia-webpack-plugin');
+const isDevBuild = process.argv.indexOf('--env.prod') < 0;
```

Comment:
Seems quite ugly. The `--env.YYY` option is there so you can `module.exports = function(env)` where `env` will be an object containing the passed in `env.YYY`.